docker-compose.dev.yml

```diff
@@ -26,17 +36,16 @@ services:
       # Temporal
       TEMPORAL__CLUSTER_URL: ${TEMPORAL__CLUSTER_URL}
       TEMPORAL__CLUSTER_QUEUE: ${TEMPORAL__CLUSTER_QUEUE}
-    restart: unless-stopped
-    networks:
-      - core
-      - temporal
+    volumes:
+      - ./tracecat:/app/tracecat
+      - core-app:/var/lib/tracecat
+
   worker:
     build:
       context: .
       dockerfile: Dockerfile.dev
-    volumes:
-      - ./tracecat:/app/tracecat
-      - core-app:/var/lib/tracecat
+    container_name: worker
```

Comment:
Any particular reason why the `container_name` was added back here? I believe we removed this when we were testing with replicating workers and you couldn't specify a name because it would autoincrement the suffix i.e. `workfer-N`

Reply:
Is mistake will remove